pep-0496-environment-markers.rst

```diff
@@ -48,14 +50,32 @@ Environment Markers attempt to provide more flexibility in a list of
 requirements by allowing the developer to list requirements that are
 specific to a particular environment.
 
+Alternatives considered
+=======================
+
+For a while, OpenStack used seperate requirements files for different
+environments - eg, ``requirements-py2.txt`` and
+``requirements-py3.txt``. However, in most cases the two files were
+near duplicates, with only one or two lines different; but maintaining
+two seperate files was more than twice as much work, as it required
+manual checking to keep the files in sync.
+
+Keeping all the requirements together in one place, with markers
+indicating which to install, has been easier to maintain.
+
 Examples
 ========
 
-Here are some examples of such markers inside a requirements.txt::
+Here is an example of such markers inside a requirements.txt
+file. This will install ``pywin32`` on Windows; ``unittest2`` on
```

Comment:
I still very very very very very much object to unittest2 being used conditionally. Its inappropriate. Please pick something where a conditional dependency is relevant - e.g. argparse. Also I suggest not using versions of Python < 2.6 in your examples.